### PR TITLE
Misc queue from 2018 (sorry, lost sources) 

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -4117,6 +4118,7 @@ contraciction->contradiction
 contracictions->contradictions
 contracition->contradiction
 contracitions->contradictions
+contracter->contractor
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add the plural

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -4216,12 +4220,18 @@ convinience->convenience
...
 convserion->conversion
+conyak->cognac
+coo->coup
 coodinate->coordinate
 coodinates->coordinates
 coodrinate->coordinate
 coodrinates->coordinates
+cooger->cougar
+cookoo->cuckoo
+coolot->culottes
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Should this not be the singular, and add the plural?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -4216,12 +4220,18 @@ convinience->convenience
...
 coonstructed->constructed
 cooporative->cooperative
+coordanate->coordinate
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add the plural

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

**Pull request merged**